test/nn/conv/BayesianGCNConv_test.py

```diff
@@ -0,0 +1,341 @@
+                                    self.prior_weight_sigma)
+
+        if self.normalize:
+            if isinstance(edge_index, Tensor):
+                cache = self._cached_edge_index
+                if cache is None:
+                    edge_index, edge_weight = gcn_norm(  # yapf: disable
+                        edge_index, edge_weight, x.size(self.node_dim),
+                        x.dtype)
+                    if self.cached:
+                        self._cached_edge_index = (edge_index, edge_weight)
+                else:
+                    edge_index, edge_weight = cache[0], cache[1]
+
+        elif isinstance(edge_index, SparseTensor):
+            cache = self._cached_adj_t
+            if cache is None:
+                edge_index = gcn_norm(  # yapf: disable
+                    edge_index, edge_weight, x.size(self.node_dim), x.dtype)
+                if self.cached:
+                    self._cached_adj_t = edge_index
+                else:
+                    edge_index = cache
+
+        sigma_weight = torch.log1p(torch.exp(self.rho_weight))
+        weight = self.mu_weight + \
+            (sigma_weight * self.eps_weight.data.normal_())
+        if return_kl:
+            kl_weight = self.kl_div(self.mu_weight, sigma_weight,
+                                    self.prior_weight_mu,
+                                    self.prior_weight_sigma)
+
+        bias = None  # Initialization of bias
+
+        if self.mu_bias is not None:
+            sigma_bias = torch.log1p(torch.exp(self.rho_bias))
+            bias = self.mu_bias + (sigma_bias * self.eps_bias.data.normal_())
+            if return_kl:
+                kl_bias = self.kl_div(self.mu_bias, sigma_bias,
+                                      self.prior_bias_mu,
+                                      self.prior_bias_sigma)
+
+        x = F.linear(x, weight, bias)
+
+        # propagate_type: (x: Tensor, edge_weight: OptTensor)
+        out = self.propagate(edge_index, x=x, edge_weight=edge_weight,
+                             size=None)
+
+        if return_kl:
+            if self.mu_bias is not None:
+                kl = kl_weight + kl_bias
+                return out, kl
+            else:
+                kl = kl_weight
+                return out
+
+    def message(self, x_j: Tensor, edge_weight: OptTensor) -> Tensor:
+        return x_j if edge_weight is None else edge_weight.view(-1, 1) * x_j
+
+    def message_and_aggregate(self, adj_t: SparseTensor, x: Tensor) -> Tensor:
+        return spmm(adj_t, x, reduce=self.aggr)
+
+
+def test_gcn_conv_with_sparse_input_feature():
+    x = torch.sparse_coo_tensor(indices=torch.tensor([[0, 0], [0, 1]]),
+                                values=torch.tensor([1., 1.]),
+                                size=torch.Size([4, 16]))
+    edge_index = torch.tensor([[0, 0, 0, 1, 2, 3], [1, 2, 3, 0, 0, 0]])
+    conv = BayesianGCNConv(16, 32)
+    out, kl = conv(x, edge_index)
+    print(out.shape, kl.detach())
```

Comment:
We don't need the print statements. Please use `assert` instead. See also `test/nn/conv/test_gcn_conv.py`.